frontend/src/views/Login/LoginView.tsx

```diff
@@ -2,23 +2,41 @@ import { RenderInput } from "components";
 import { Link } from "react-router-dom";
 import styles from "./Login.module.scss";
 import { LoginInputs } from "interfaces/index";
-import { User } from "interfaces";
+import React from "react";
 
 interface LoginViewProps {
     onFormSubmit: (e: React.FormEvent) => void;
     onInputChange: (e: React.ChangeEvent) => void;
     inputsArray: LoginInputs[];
     errorMessage: string | null;
-    loggedUser: User | null;
 }
 
+const url = "http://localhost:8000/user/logout";
+
 export const LoginView = ({
     onFormSubmit,
     onInputChange,
     inputsArray,
     errorMessage,
-    loggedUser,
 }: LoginViewProps) => {
+    const logoutUser = async (e: React.MouseEvent) => {
+        e.preventDefault();
+
+        const response = await fetch(url, {
```

Comment:
suggestion: move this inside of an redux action, it will clean up the component code a bit. You can also create a sort of a AuthService which would expose the methods and do the fetch stuff inside of them so the flow would look like this

[component] clickHandler -> dispatch(LogoutAction) -> [redux slice, inside thunk] await AuthService.logout()